shopinvader/services/cart.py

```diff
@@ -515,6 +515,7 @@ def _prepare_cart_item(self, params, cart):
             {
                 "product_uom_qty": params["item_qty"],
                 "order_id": cart.id,
+                "sequence": max(cart.order_line.mapped("sequence") or [0]) + 1,
```

Comment:
```suggestion
                "sequence": max(cart.order_line.mapped("sequence"), default=0) + 1,
```
This also works

Reply:
Thanks for the tips! It's cleaner indeed